composer.json

```diff
@@ -54,7 +54,7 @@
     ],
     "require": {
         "php": ">=7.4 <8.2",
-        "atk4/data": "dev-develop"
+        "atk4/data": "dev-norm_id_array_pers as dev-develop"
```

Comment:
TODO remove before merge

@ibelar ui is failing with this data PR, can you please fix it here? It seems that a field name is set as a field value somewhere and not passing numeric validation.